### DISCUSSION TITLE
Problem with the new version of Pluto in Safari

### DISCUSSION THREAD

**Discussion by davidgmiguez** [fonsp/Pluto.jl#2897] [Pluto development] [answered]
Dear all, 
 in the new verison of pluto in safari i have overlapping between buttons that control cell behavior. This makes pluto mainly unusable. I am including a snapshot of the problem. 

<img width="312" alt="Screenshot 2024-04-19 at 08 28 01" src="https://github.com/fonsp/Pluto.jl/assets/46992715/da57def7-e27c-4994-bb87-c006e1381fd8">

I am attaching an image to show how the "eye" icon and the button that disables a cell appear overlapped. 

Also, it used to work very well when editing and selecting text inside a markdown cell, but now it does not select the text where you click. Thanks a lot for your help.

**Comment by fonsp** [comment, marked as answer]
Thanks! This was fixed in https://github.com/fonsp/Pluto.jl/issues/2825
